lib/elastic_apm/transport/filters/secrets_filter.rb

```diff
@@ -30,6 +31,7 @@ def initialize(config)
         def call(payload)
           strip_from! payload.dig(:transaction, :context, :request, :headers)
           strip_from! payload.dig(:transaction, :context, :request, :env)
+          strip_from! payload.dig(:transaction, :context, :request, :cookies)
```

Comment:
Just trying to understand what's going on: is this going to strip secrets from `request.cookies`? I can't see a test for this.

Reply:
Yes, that's correct. There's not a specific test for this field but there are for the others and the implementation is the same:
https://github.com/elastic/apm-agent-ruby/blob/master/spec/elastic_apm/transport/filters/secrets_filter_spec.rb

Reply:
OK. Generally I'd like to see full coverage for sensitive things like this, but I think we should address this with a functional test for all agents.